src/components/HoldersForm.tsx

```diff
@@ -45,7 +102,23 @@ export const HoldersForm = ({ setTokenId, setMinAmount, setData, setShouldFetch,
                 <FormItem>
                   <FormLabel>{dictionary.tokenId}</FormLabel>
                   <FormControl>
-                    <Input placeholder={dictionary.exampleTokenId} {...field} />
+                    <>
+                      <Input
+                        placeholder={dictionary.exampleTokenId}
+                        {...field}
+                        onChange={(e) => {
+                          setTokenIdValue(e.target.value);
+                          if (!e.target.value || !/^0\.0\.\d*$/.test(e.target.value)) setAccountName(undefined);
+                          field.onChange(e);
+                        }}
```

Comment:
Maybe it will be a good idea to separate this code to another function and leave a comment for `!/^0\.0\.\d*$/`?

Reply:
Done